fzmp

```diff
@@ -189,6 +191,7 @@ filter_by_playlist() {
     '<') mpc -q prev; filter_by_playlist ;;
     'ctrl-d') [[ -n "${choice[1]}" ]] && mpc -q del "${choice[@]:1}"; filter_by_playlist ;;
     'enter') [[ -n "${choice[1]}" ]] && mpc -q play "${choice[@]:1}"; filter_by_playlist ;;
+    'del') [[ -n "${choice[1]}" ]] && mpc -q del "${choice[@]:1}"; filter_by_playlist ;;
```

Comment:
Adding an extra key for deletion should be done on the same line as the other, so L192 should be  `'ctrl-d'|'del') ...`